definitions/npm/commander_v2.x.x/flow_>=0.28.x/commander_v2.x.x.js

```diff
@@ -0,0 +1,276 @@
+declare class $npm$commander$Command extends events$EventEmitter {
+  /**
+   * Initialize a new `Command`.
+   *
+   * @param {String} name
+   * @api public
+   */
+  constructor(name?: string): $npm$commander$Command;
+
+  /**
+   * Add command `name`.
+   *
+   * The `.action()` callback is invoked when the
+   * command `name` is specified via __ARGV__,
+   * and the remaining arguments are applied to the
+   * function for access.
+   *
+   * When the `name` is "*" an un-matched command
+   * will be passed as the first arg, followed by
+   * the rest of __ARGV__ remaining.
+   *
+   * Examples:
+   *
+   *      program
+   *        .version('0.0.1')
+   *        .option('-C, --chdir <path>', 'change the working directory')
+   *        .option('-c, --config <path>', 'set config path. defaults to ./deploy.conf')
+   *        .option('-T, --no-tests', 'ignore test hook')
+   *
+   *      program
+   *        .command('setup')
+   *        .description('run remote setup commands')
+   *        .action(function(){
+   *          console.log('setup');
+   *        });
+   *
+   *      program
+   *        .command('exec <cmd>')
+   *        .description('run the given remote command')
+   *        .action(function(cmd){
+   *          console.log('exec "%s"', cmd);
+   *        });
+   *
+   *      program
+   *        .command('*')
+   *        .description('deploy the given env')
+   *        .action(function(env){
+   *          console.log('deploying "%s"', env);
+   *        });
+   *
+   *      program.parse(process.argv);
+   *
+   * @param {String} name
+   * @param {String} [desc]
+   * @param {Mixed} [opts]
+   * @return {Command} the new command
+   * @api public
+   */
+  command(name: string, desc?: string, opts?: any): $npm$commander$Command;
```

Comment:
At minimum, `opts` should be an optional `Object` probably, right?

(Ideally there'd be a full-on type for it though... If you don't want to write one right now, mind adding a `TODO` for it?)